instrumented/nightly-tests/src/main/kotlin/com/datadog/android/nightly/activities/ViewTrackingFragmentActivity.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.nightly.activities
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.datadog.android.nightly.R
+import com.datadog.android.nightly.fragments.ViewTrackingFragment
+
+internal class ViewTrackingFragmentActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.tracking_strategy_fragment_activity)
+        supportFragmentManager
+            .beginTransaction()
+            .add(R.id.fragment_container, ViewTrackingFragment::class.java, null)
+            .commit()
```

Comment:
should we do `commitNow` maybe to get rid of async nature here? or it won't impact the test?

Reply:
I don't think it will impact this test as Espresso waits until MainThread Looper is idled anyway. Also I would rather have a more common use case as I think users prefer the `commit` instead the `commitNow` or not ?

Reply:
ok, indeed wait for idle in Espresso makes sense, we can use `commit`